<?xml version="1.0" encoding="UTF-8"?>
<!-- -*- mode: XML -*- -->
<sdf version='1.4'>
	<model name='colorado'>
		<link name='body'>
			<pose>0 0 0 0 0 0</pose>
			<gravity>1</gravity>
			<self_collide>1</self_collide>
			<inertial>
				<pose>0.00 0.00 1 0 0 0</pose>
				<mass>2000</mass>
				<inertia>
					<ixx>500</ixx>
					<ixy>0.00</ixy>
					<ixz>0.00</ixz>
					<iyy>2000</iyy>
					<iyz>0.00</iyz>
					<izz>2000</izz>
				</inertia>
			</inertial>
			<collision name='body_collision_base_mid'>
				<pose>-0.17975 0 1.20715 0 0 0</pose>
				<geometry>
					<box>
						<size>1.934 2.392 1.438</size>
					</box>
				</geometry>
			</collision>
			<collision name='body_collision_base_hood'>
				<pose>1.65953 0 1.20715 0 0.09472 0</pose>
				<geometry>
					<box>
						<size>1.622 2.414 0.1</size>
					</box>
				</geometry>
			</collision>
			<collision name='body_collision_base_back'>
				<pose>-2.59851 0 1.08547 0 0 0</pose>
				<geometry>
					<box>
						<size>0.158 2.46 0.591</size>
					</box>
				</geometry>
			</collision>
			<collision name='body_collision_base_back_top'>
				<pose>-1.93823 0 1.56472 0 -0.33794 0</pose>
				<geometry>
					<box>
						<size>1.384 2.460 0.269</size>
					</box>
				</geometry>
			</collision>
			<visual name='body_visual'>
				<pose>0 0 0 0 0 0</pose>
				<geometry>
					<mesh>
						<scale>1 1 1</scale>
						<uri>model://../Assets/colorado_body.dae</uri>
					</mesh>
				</geometry>
			</visual>
		</link>
		

<!--Suspension setup-->
<!--Front left wheel-->
<link name="suspension_main_left_1">
<pose>1.6 0.27504 0.65450 0 0 0</pose>
  <inertial>
    <inertia>
      <ixx>0.01</ixx>
      <ixy>0.0</ixy>
      <ixz>0.0</ixz>
      <iyy>0.01</iyy>
      <iyz>0.0</iyz>
      <izz>0.01</izz>
    </inertia>
     <mass>50</mass>
  </inertial>
</link>
<link name="suspension_holder_left_1">
<pose>1.6 0.3666 0.8654 1.09558 0 0</pose>
  <inertial>
    <inertia>
      <ixx>0.01</ixx>
      <ixy>0.0</ixy>
      <ixz>0.0</ixz>
      <iyy>0.01</iyy>
      <iyz>0.0</iyz>
      <izz>0.01</izz>
    </inertia>
     <mass>50</mass>
  </inertial>
  <visual name='visual'>
    <pose>0 0 -0.25 0 0 0</pose>
    <geometry>
      <cylinder>
        <radius>0.025</radius>
        <length>0.5</length>
      </cylinder>
    </geometry>
    <material><ambient>0.1 0.1 0.1 1</ambient><diffuse>0.1 0.1 0.1 1</diffuse><specular>1 1 1 1</specular><emissive>0 0 0 1</emissive></material>
  </visual>
</link>
<link name="suspension_left_1">
<pose>1.6 0.9 0.391 0 0 0</pose>
  <inertial>
    <inertia>
      <ixx>0.01</ixx>
      <ixy>0.0</ixy>
      <ixz>0.0</ixz>
      <iyy>0.01</iyy>
      <iyz>0.0</iyz>
      <izz>0.01</izz>
    </inertia>
     <mass>50</mass>
  </inertial>
  <visual name='visual'>
    <pose>0 0 0 0 0 0</pose>
    <geometry>
      <cylinder>
        <radius>0.025</radius>
        <length>0.1</length>
      </cylinder>
    </geometry>
  </visual>
</link>
<joint name="spring_left_1" type="revolute">
  <parent>body</parent>
  <child>suspension_main_left_1</child>
  <pose>0.0 0.0 0.0 0.0 0.0 0.0</pose>
  <axis>
    <xyz>1 0 0</xyz>
  </axis>
</joint>
<joint name="suspension_holder_joint_left_1" type="revolute">
  <parent>body</parent>
  <child>suspension_holder_left_1</child>
  <pose>0.0 0.0 0.0 0.0 0.0 0.0</pose>
  <axis>
    <xyz>1 0 0</xyz>
  </axis>
</joint>
<joint name="suspension_holder_wheel_joint_left_1" type="revolute">
  <parent>suspension_holder_left_1</parent>
  <child>suspension_left_1</child>
  <pose>0.0 0.0 0.21 0.0 0.0 0.0</pose>
  <axis>
    <xyz>1 0 0</xyz>
    <limit>
      <lower>-0.5</lower>
      <upper>0.5</upper>
    </limit>
  </axis><!--Front left wheel-->
</joint>
<joint name="suspension_main_wheel_joint_left_1" type="revolute">
  <parent>suspension_main_left_1</parent>
  <child>suspension_left_1</child>
  <pose>0.0 0.0 -0.11 0.0 0.0 0.0</pose>
  <axis>
    <xyz>1 0 0</xyz>
  </axis>
</joint>
<!--Back left wheel-->
<link name="suspension_main_left_2">
<pose>-1.6 0.27504 0.65450 0 0 0</pose>
  <inertial>
    <inertia>
      <ixx>0.01</ixx>
      <ixy>0.0</ixy>
      <ixz>0.0</ixz>
      <iyy>0.01</iyy>
      <iyz>0.0</iyz>
      <izz>0.01</izz>
    </inertia>
     <mass>50</mass>
  </inertial>

</link>
<link name="suspension_holder_left_2">
<pose>-1.6 0.3666 0.8654 1.09558 0 0</pose>
  <inertial>
    <inertia>
      <ixx>0.01</ixx>
      <ixy>0.0</ixy>
      <ixz>0.0</ixz>
      <iyy>0.01</iyy>
      <iyz>0.0</iyz>
      <izz>0.01</izz>
    </inertia>
     <mass>50</mass>
  </inertial>
  <visual name='visual'>
    <pose>0 0 -0.25 0 0 0</pose>
    <geometry>
      <cylinder>
        <radius>0.025</radius>
        <length>0.5</length>
      </cylinder>
    </geometry>
    <material><ambient>0.1 0.1 0.1 1</ambient><diffuse>0.1 0.1 0.1 1</diffuse><specular>1 1 1 1</specular><emissive>0 0 0 1</emissive></material>
  </visual>
</link>
<link name="suspension_left_2">
<pose>-1.6 0.9 0.391 0 0 0</pose>
  <inertial>
    <inertia>
      <ixx>0.01</ixx>
      <ixy>0.0</ixy>
      <ixz>0.0</ixz>
      <iyy>0.01</iyy>
      <iyz>0.0</iyz>
      <izz>0.01</izz>
    </inertia>
     <mass>50</mass>
  </inertial>
  <visual name='visual'>
    <pose>0 0 0 0 0 0</pose>
    <geometry>
      <cylinder>
        <radius>0.025</radius>
        <length>0.1</length>
      </cylinder>
    </geometry>
  </visual>
</link>
<joint name="spring_left_2" type="revolute">
  <parent>body</parent>
  <child>suspension_main_left_2</child>
  <pose>0.0 0.0 0.0 0.0 0.0 0.0</pose>
  <axis>
    <xyz>1 0 0</xyz>
  </axis>
</joint>
<joint name="suspension_holder_joint_left_2" type="revolute">
  <parent>body</parent>
  <child>suspension_holder_left_2</child>
  <pose>0.0 0.0 0.0 0.0 0.0 0.0</pose>
  <axis>
    <xyz>1 0 0</xyz>
  </axis>
</joint>
<joint name="suspension_holder_holder_wheel_joint_left_2" type="revolute">
  <parent>suspension_holder_left_2</parent>
  <child>suspension_left_2</child>
  <pose>0.0 0.0 0.21 0.0 0.0 0.0</pose>
  <axis>
    <xyz>1 0 0</xyz>
    <limit>
      <lower>-0.5</lower>
      <upper>0.5</upper>
    </limit>
  </axis>
</joint>
<joint name="suspension_main_wheel_joint_left_2" type="revolute">
  <parent>suspension_main_left_2</parent>
  <child>suspension_left_2</child>
  <pose>0.0 0.0 -0.11 0.0 0.0 0.0</pose>
  <axis>
    <xyz>1 0 0</xyz>
  </axis>
</joint>
<!--Front right wheel-->
<link name="suspension_main_right_1">
<pose>1.6 -0.27504 0.65450 0 0 0</pose>
  <inertial>
    <inertia>
      <ixx>0.01</ixx>
      <ixy>0.0</ixy>
      <ixz>0.0</ixz>
      <iyy>0.01</iyy>
      <iyz>0.0</iyz>
      <izz>0.01</izz>
    </inertia>
     <mass>50</mass>
  </inertial>
</link>
<link name="suspension_holder_right_1">
<pose>1.6 -0.3666 0.8654 -1.09558 0 0</pose>
  <inertial>
    <inertia>
      <ixx>0.01</ixx>
      <ixy>0.0</ixy>
      <ixz>0.0</ixz>
      <iyy>0.01</iyy>
      <iyz>0.0</iyz>
      <izz>0.01</izz>
    </inertia>
     <mass>50</mass>
  </inertial>
  <visual name='visual'>
    <pose>0 0 -0.25 0 0 0</pose>
    <geometry>
      <cylinder>
        <radius>0.025</radius>
        <length>0.5</length>
      </cylinder>
    </geometry>
    <material><ambient>0.1 0.1 0.1 1</ambient><diffuse>0.1 0.1 0.1 1</diffuse><specular>1 1 1 1</specular><emissive>0 0 0 1</emissive></material>
  </visual>
</link>
<link name="suspension_right_1">
<pose>1.6 -0.9 0.391 0 0 0</pose>
  <inertial>
    <inertia>
      <ixx>0.01</ixx>
      <ixy>0.0</ixy>
      <ixz>0.0</ixz>
      <iyy>0.01</iyy>
      <iyz>0.0</iyz>
      <izz>0.01</izz>
    </inertia>
     <mass>50</mass>
  </inertial>
  <visual name='visual'>
    <pose>0 0 0 0 0 0</pose>
    <geometry>
      <cylinder>
        <radius>0.025</radius>
        <length>0.1</length>
      </cylinder>
    </geometry>
  </visual>
</link>
<joint name="spring_right_1" type="revolute">
  <parent>body</parent>
  <child>suspension_main_right_1</child>
  <pose>0.0 0.0 0.0 0.0 0.0 0.0</pose>
  <axis>
    <xyz>-1 0 0</xyz>
  </axis>
</joint>
<joint name="suspension_holder_joint_right_1" type="revolute">
  <parent>body</parent>
  <child>suspension_holder_right_1</child>
  <pose>0.0 0.0 0.0 0.0 0.0 0.0</pose>
  <axis>
    <xyz>1 0 0</xyz>
  </axis>
</joint>
<joint name="suspension_holder_wheel_joint_right_1" type="revolute">
  <parent>suspension_holder_right_1</parent>
  <child>suspension_right_1</child>
  <pose>0.0 0.0 0.21 0.0 0.0 0.0</pose>
  <axis>
    <xyz>1 0 0</xyz>
    <limit>
      <lower>-0.5</lower>
      <upper>0.5</upper>
    </limit>
  </axis>
</joint>
<joint name="suspension_main_wheel_joint_right_1" type="revolute">
  <parent>suspension_main_right_1</parent>
  <child>suspension_right_1</child>
  <pose>0.0 0.0 -0.11 0.0 0.0 0.0</pose>
  <axis>
    <xyz>1 0 0</xyz>
  </axis>
</joint>
<!--Back right wheel-->
<link name="suspension_main_right_2">
<pose>-1.6 -0.27504 0.65450 0 0 0</pose>
  <inertial>
    <inertia>
      <ixx>0.01</ixx>
      <ixy>0.0</ixy>
      <ixz>0.0</ixz>
      <iyy>0.01</iyy>
      <iyz>0.0</iyz>
      <izz>0.01</izz>
    </inertia>
     <mass>50</mass>
  </inertial>
</link>
<link name="suspension_holder_right_2">
<pose>-1.6 -0.3666 0.8654 -1.09558 0 0</pose>
  <inertial>
    <inertia>
      <ixx>0.01</ixx>
      <ixy>0.0</ixy>
      <ixz>0.0</ixz>
      <iyy>0.01</iyy>
      <iyz>0.0</iyz>
      <izz>0.01</izz>
    </inertia>
     <mass>50</mass>
  </inertial>
  <visual name='visual'>
    <pose>0 0 -0.25 0 0 0</pose>
    <geometry>
      <cylinder>
        <radius>0.025</radius>
        <length>0.5</length>
      </cylinder>
    </geometry>
    <material><ambient>0.1 0.1 0.1 1</ambient><diffuse>0.1 0.1 0.1 1</diffuse><specular>1 1 1 1</specular><emissive>0 0 0 1</emissive></material>
  </visual>
</link>
<link name="suspension_right_2">
<pose>-1.6 -0.9 0.391 0 0 0</pose>
  <inertial>
    <inertia>
      <ixx>0.01</ixx>
      <ixy>0.0</ixy>
      <ixz>0.0</ixz>
      <iyy>0.01</iyy>
      <iyz>0.0</iyz>
      <izz>0.01</izz>
    </inertia>
     <mass>50</mass>
  </inertial>
  <visual name='visual'>
    <pose>0 0 0 0 0 0</pose>
    <geometry>
      <cylinder>
        <radius>0.025</radius>
        <length>0.1</length>
      </cylinder>
    </geometry>
  </visual>
</link>
<joint name="spring_right_2" type="revolute">
  <parent>body</parent>
  <child>suspension_main_right_2</child>
  <pose>0.0 0.0 0.0 0.0 0.0 0.0</pose>
  <axis>
    <xyz>-1 0 0</xyz>
  </axis>
</joint>
<joint name="suspension_holder_joint_right_2" type="revolute">
  <parent>body</parent>
  <child>suspension_holder_right_2</child>
  <pose>0.0 0.0 0.0 0.0 0.0 0.0</pose>
  <axis>
    <xyz>1 0 0</xyz>
  </axis>
</joint>
<joint name="suspension_holder_wheel_joint_right_2" type="revolute">
  <parent>suspension_holder_right_2</parent>
  <child>suspension_right_2</child>
  <pose>0.0 0.0 0.21 0.0 0.0 0.0</pose>
  <axis>
    <xyz>1 0 0</xyz>
    <limit>
      <lower>-0.5</lower>
      <upper>0.5</upper>
    </limit>
  </axis>
</joint>
<joint name="suspension_main_wheel_joint_right_2" type="revolute">
  <parent>suspension_main_right_2</parent>
  <child>suspension_right_2</child>
  <pose>0.0 0.0 -0.11 0.0 0.0 0.0</pose>
  <axis>
    <xyz>1 0 0</xyz>
  </axis>
</joint>
<!--End of suspension setup-->

<!--steering setup-->
<link name="steering_left_1">
  <pose>1.6 0.9 0.391 0 0 0</pose>
  <inertial>
    <inertia>
      <ixx>0.01</ixx>
      <ixy>0.0</ixy>
      <ixz>0.0</ixz>
      <iyy>0.01</iyy>
      <iyz>0.0</iyz>
      <izz>0.01</izz>
    </inertia>
    <mass>50</mass>
  </inertial>
  <visual name='visual'>
    <pose>0 0 0 0 0 3.14159265359</pose>
    <geometry>
      <cylinder>
        <radius>0.02</radius>
        <length>0.3</length>
      </cylinder>
    </geometry>
    <material><ambient>0.2 0.2 0.2 1</ambient><diffuse>0.1 0.1 0.1 1</diffuse><specular>1 1 1 1</specular><emissive>0 0 0 1</emissive></material>
  </visual>
</link>
<joint name="steering_joint_left_1" type="revolute">
  <parent>suspension_left_1</parent>
  <child>steering_left_1</child>
  <pose>0.0 0.0 0.0 0.0 0.0 0.0</pose>
  <axis>
    <dynamics>
      <damping>1</damping>
    </dynamics>
    <limit>
      <lower>-0.61</lower>
      <upper>0.61</upper>
    </limit>
    <xyz>0 0 1</xyz>
  </axis>
</joint>
<link name="steering_right_1">
  <pose>1.6 -0.9 0.391 0 0 0</pose>
  <inertial>
    <inertia>
      <ixx>0.01</ixx>
      <ixy>0.0</ixy>
      <ixz>0.0</ixz>
      <iyy>0.01</iyy>
      <iyz>0.0</iyz>
      <izz>0.01</izz>
    </inertia>
    <mass>50</mass>
  </inertial>
  <visual name='visual'>
    <pose>0 0 0 0 0 0</pose>
    <geometry>
      <cylinder>
        <radius>0.02</radius>
        <length>0.3</length>
      </cylinder>
    </geometry>
    <material><ambient>0.2 0.2 0.2 1</ambient><diffuse>0.1 0.1 0.1 1</diffuse><specular>1 1 1 1</specular><emissive>0 0 0 1</emissive></material>
  </visual>
</link>
<joint name="steering_joint_right_1" type="revolute">
  <parent>suspension_right_1</parent>
  <child>steering_right_1</child>
  <pose>0.0 0.0 0.0 0.0 0.0 0.0</pose>
  <axis>
    <dynamics>
      <damping>1</damping>
    </dynamics>
    <limit>
      <lower>-0.61</lower>
      <upper>0.61</upper>
    </limit>
    <xyz>0 0 1</xyz>
  </axis>
</joint>
<!--End of steering setup-->

<!--wheels Setup-->
<link name='left_wheel_1'>
	<pose>1.6 0.9 0.391 0 0 0</pose>
	<inertial>
		<pose>0 0 0 0 0 0</pose>
		<mass>80</mass>
		<inertia>
			<ixx>5.9</ixx>
			<ixy>0</ixy>
			<ixz>0</ixz>
			<iyy>9.88</iyy>
			<iyz>0</iyz>
			<izz>5.9</izz>
		</inertia>
	</inertial>
	<collision name='wheel_col'>
		<pose>0 0 0 1.57079633 0 0</pose>
		<geometry>
			<cylinder>
				<length>0.1</length>
				<radius>0.388</radius>
			</cylinder>
		</geometry>
	</collision>
		<collision name='wheel_col'>
		<pose>0 0 0 1.57079633 0 0</pose>
		<geometry>
			<cylinder>
				<length>0.2</length>
				<radius>0.387</radius>
			</cylinder>
		</geometry>
	</collision>
	<collision name='wheel_col'>
		<pose>0 0 0 1.57079633 0 0</pose>
		<geometry>
			<cylinder>
				<length>0.24</length>
				<radius>0.385</radius>
			</cylinder>
		</geometry>
	</collision>
	<collision name='wheel_col'>
		<pose>0 0 0 1.57079633 0 0</pose>
		<geometry>
			<cylinder>
				<length>0.254</length>
				<radius>0.38</radius>
			</cylinder>
		</geometry>
	</collision>
	<visual name='left_wheel'>
		<pose>0 0 0 0 0 3.1469265</pose>
		<geometry>
			<mesh>
				<uri>model://../Assets/colorado_wheel.dae</uri>
			</mesh>
		</geometry>
	</visual>
	<gravity>1</gravity>
	<self_collide>0</self_collide>
</link>
<joint name='left_wheel_1' type='revolute'>
	<child>left_wheel_1</child>
	<parent>steering_left_1</parent>
	<axis>
		<xyz>0 1 0</xyz>
	</axis>
</joint>
<link name='left_wheel_2'>
	<pose>-1.6 0.9 0.391 0 0 0</pose>
	<inertial>
		<pose>0 0 0 0 0 0</pose>
		<mass>80</mass>
		<inertia>
			<ixx>5.9</ixx>
			<ixy>0</ixy>
			<ixz>0</ixz>
			<iyy>9.88</iyy>
			<iyz>0</iyz>
			<izz>5.9</izz>
		</inertia>
	</inertial>
	<collision name='wheel_col'>
		<pose>0 0 0 1.57079633 0 0</pose>
		<geometry>
			<cylinder>
				<length>0.1</length>
				<radius>0.388</radius>
			</cylinder>
		</geometry>
	</collision>
		<collision name='wheel_col'>
		<pose>0 0 0 1.57079633 0 0</pose>
		<geometry>
			<cylinder>
				<length>0.2</length>
				<radius>0.387</radius>
			</cylinder>
		</geometry>
	</collision>
	<collision name='wheel_col'>
		<pose>0 0 0 1.57079633 0 0</pose>
		<geometry>
			<cylinder>
				<length>0.24</length>
				<radius>0.385</radius>
			</cylinder>
		</geometry>
	</collision>
	<collision name='wheel_col'>
		<pose>0 0 0 1.57079633 0 0</pose>
		<geometry>
			<cylinder>
				<length>0.254</length>
				<radius>0.38</radius>
			</cylinder>
		</geometry>
	</collision>
	<visual name='left_wheel'>
		<pose>0 0 0 0 0 3.1469265</pose>
		<geometry>
			<mesh>
				<uri>model://../Assets/colorado_wheel.dae</uri>
			</mesh>
		</geometry>
	</visual>
	<gravity>1</gravity>
	<self_collide>0</self_collide>
</link>
<joint name='left_wheel_2' type='revolute'>
	<child>left_wheel_2</child>
	<parent>suspension_left_2</parent>
	<axis>
		<xyz>0 1 0</xyz>
	</axis>
</joint>
<link name='right_wheel_1'>
	<pose>1.6 -0.9 0.391 0 0 0</pose>
	<inertial>
		<pose>0 0 0 0 0 0</pose>
		<mass>80</mass>
		<inertia>
			<ixx>5.9</ixx>
			<ixy>0</ixy>
			<ixz>0</ixz>
			<iyy>9.88</iyy>
			<iyz>0</iyz>
			<izz>5.9</izz>
		</inertia>
	</inertial>
	<collision name='wheel_col'>
		<pose>0 0 0 1.57079633 0 0</pose>
		<geometry>
			<cylinder>
				<length>0.1</length>
				<radius>0.388</radius>
			</cylinder>
		</geometry>
	</collision>
		<collision name='wheel_col'>
		<pose>0 0 0 1.57079633 0 0</pose>
		<geometry>
			<cylinder>
				<length>0.2</length>
				<radius>0.387</radius>
			</cylinder>
		</geometry>
	</collision>
	<collision name='wheel_col'>
		<pose>0 0 0 1.57079633 0 0</pose>
		<geometry>
			<cylinder>
				<length>0.24</length>
				<radius>0.385</radius>
			</cylinder>
		</geometry>
	</collision>
	<collision name='wheel_col'>
		<pose>0 0 0 1.57079633 0 0</pose>
		<geometry>
			<cylinder>
				<length>0.254</length>
				<radius>0.38</radius>
			</cylinder>
		</geometry>
	</collision>
	<visual name='right_wheel'>
		<pose>0 0 0 0 0 0</pose>
		<geometry>
			<mesh>
				<uri>model://../Assets/colorado_wheel.dae</uri>
			</mesh>
		</geometry>
	</visual>
	<gravity>1</gravity>
	<self_collide>0</self_collide>
</link>
<joint name='right_wheel_1' type='revolute'>
	<child>right_wheel_1</child>
	<parent>steering_right_1</parent>
	<axis>
		<xyz>0 1 0</xyz>
	</axis>
</joint>
<link name='right_wheel_2'>
	<pose>-1.6 -0.9 0.391 0 0 0</pose>
	<inertial>
		<pose>0 0 0 0 0 0</pose>
		<mass>80</mass>
		<inertia>
			<ixx>5.9</ixx>
			<ixy>0</ixy>
			<ixz>0</ixz>
			<iyy>9.88</iyy>
			<iyz>0</iyz>
			<izz>5.9</izz>
		</inertia>
	</inertial>
	<collision name='wheel_col'>
		<pose>0 0 0 1.57079633 0 0</pose>
		<geometry>
			<cylinder>
				<length>0.1</length>
				<radius>0.388</radius>
			</cylinder>
		</geometry>
	</collision>
		<collision name='wheel_col'>
		<pose>0 0 0 1.57079633 0 0</pose>
		<geometry>
			<cylinder>
				<length>0.2</length>
				<radius>0.387</radius>
			</cylinder>
		</geometry>
	</collision>
	<collision name='wheel_col'>
		<pose>0 0 0 1.57079633 0 0</pose>
		<geometry>
			<cylinder>
				<length>0.24</length>
				<radius>0.385</radius>
			</cylinder>
		</geometry>
	</collision>
	<collision name='wheel_col'>
		<pose>0 0 0 1.57079633 0 0</pose>
		<geometry>
			<cylinder>
				<length>0.254</length>
				<radius>0.38</radius>
			</cylinder>
		</geometry>
	</collision>
	<visual name='right_wheel'>
		<pose>0 0 0 0 0 0</pose>
		<geometry>
			<mesh>
				<uri>model://../Assets/colorado_wheel.dae</uri>
			</mesh>
		</geometry>
	</visual>
	<gravity>1</gravity>
	<self_collide>0</self_collide>
</link>
<joint name='right_wheel_2' type='revolute'>
	<child>right_wheel_2</child>
	<parent>suspension_right_2</parent>
	<axis>
		<xyz>0 1 0</xyz>
	</axis>
</joint>
<!--End of wheels setup-->

		<!--<plugin name="joint_state_publisher" filename="libgazebo_ros_joint_state_publisher.so"><robotNamespace>/Sahar</robotNamespace><jointName>left_wheel_1, </jointName><updateRate>50.0</updateRate><alwaysOn>true</alwaysOn></plugin>-->
	</model>
</sdf>